finality-aleph/src/data_io/data_provider.rs

```diff
@@ -366,8 +364,13 @@ mod tests {
             refresh_interval: REFRESH_INTERVAL,
         };
 
-        let (chain_tracker, data_provider) =
-            ChainTracker::new(select_chain, client, session_boundaries, config, None);
+        let (chain_tracker, data_provider) = ChainTracker::new(
+            select_chain,
+            client,
+            session_boundaries,
+            config,
+            Metrics::register(None, |_| {}),
```

Comment:
if you will follow my earlier suggestion then this would be simply Metrics::new(None).

Reply:
Yup, now it's a little less confusing.